tests/foreman/cli/test_provisioningtemplate.py

```diff
@@ -197,80 +153,41 @@ def test_positive_create_locked():
 
     :CaseImportance: Medium
     """
-    new_template = make_template({'locked': 'true', 'name': gen_string('alpha')})
+    new_template = module_target_sat.cli_factory.make_template(
+        {'locked': 'true', 'name': gen_string('alpha')}
+    )
     assert new_template['locked'] == 'yes'
 
 
-@pytest.mark.tier2
-def test_positive_create_with_org(module_org):
-    """Check if Template with Organization can be created
-
-    :id: 5de5ca76-1a39-46ac-8dd4-5d41b4b49076
-
-    :expectedresults: Template is created and new Organization has been
-        assigned
-
-    :CaseImportance: Medium
-    """
-    new_template = make_template({'name': gen_string('alpha'), 'organization-ids': module_org.id})
-    assert module_org.name in new_template['organizations']
-
-
 @pytest.mark.tier2
 @pytest.mark.upgrade
-def test_positive_add_os_by_id(module_os_with_minor):
-    """Check if operating system can be added to a template
+def test_positive_add_remove_os_by_id(module_target_sat, module_os_with_minor):
+    """Check if operating system can be added and removed to a template
 
     :id: d9f481b3-9757-4208-b451-baf4792d4d70
```

Comment:
Can you add the BZ of test_positive_remove_os_by_id in this test case since you merged these two.

Reply:
Good catch, but I intensionally missed it as the BZ linked wasn't related to the test